### PR TITLE
readme: use prefixed method names

### PR DESCRIPTION
**Pull request by davidchambers**
These changes make the document clearer but uglier. Clarity trumps beauty, in this context.

:thumbsup: / :thumbsdown: ?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -474,82 +471,88 @@ Given a value `f`, one can access its type representative via the
...
 
-1. `x.ap(f.alt(g))` is equivalent to `x.ap(f).alt(x.ap(g))` (distributivity)
-2. `x.ap(A.zero())` is equivalent to `A.zero()` (annihilation)
+1. `x['fantasy-land/ap'](f['fantasy-land/alt'](g))` is equivalent to `x['fantasy-land/ap'](f)['fantasy-land/alt'](x['fantasy-land/ap'](g))` (distributivity)
+2. `x['fantasy-land/ap'](A['fantasy-land/zero']())` is equivalent to `A['fantasy-land/zero']()` (annihilation)
 
 ### Foldable
 
-1. `u.reduce` is equivalent to `u.reduce((acc, x) => acc.concat([x]), []).reduce`
+1. `u['fantasy-land/reduce']` is equivalent to `u['fantasy-land/reduce']((acc, x) => acc.concat([x]), []).reduce`
```
Comment (davidchambers) [inline README.md]: We really do want `.concat` and `.reduce` here because we're referring to `Array#concat` and `Array#reduce`.

File: README.md [hunk 2]
```diff
@@ -163,309 +136,333 @@ a `constructor` property which is a reference to the type representative.
...
 
-1. `a.equals(a) === true` (reflexivity)
-2. `a.equals(b) === b.equals(a)` (symmetry)
-3. If `a.equals(b)` and `b.equals(c)`, then `a.equals(c)` (transitivity)
+1. `a['fantasy-land/equals'](a) === true` (reflexivity)
+2. `a['fantasy-land/equals'](b) === b['fantasy-land/equals'](a)` (symmetry)
+3. If `a['fantasy-land/equals'](b)` and `b['fantasy-land/equals'](c)`, then `a['fantasy-land/equals'](c)` (transitivity)
 
-#### `equals` method
+<a name="equals-method"></a>
```
Comment (davidchambers) [inline README.md]: As far as I'm aware GitHub does not provide a way to customize a heading's fragment identifier.

**Comment by xaviervia** [comment]
@davidchambers this makes a lot of sense to me, but as an observer tracking what people think about FL, I think it might be a bit of a public relations problem. As it is today, a lot of libraries are _de facto_ partially compatible with FL. After the change, most of them will no longer be (`ramda` comes to mind).

**Comment by davidchambers** [comment]
@xaviervia, the spec has required prefixed method names since #146 was released in [v1.0.0][1] back in 2016. This pull request just updates the documentation to make this existing requirement apparent.

_p.s. I'm not holding my breath waiting for Ramda to fully support Fantasy Land._


[1]: https://github.com/fantasyland/fantasy-land/releases/tag/v1.0.0

**Comment by xaviervia** [comment]
Right! Then it would be great to do this, since I was not aware of that
requirement and I might not have been the only one missing that

**Pull request merged**